app/webpacker/styles/header_overrides.scss

```diff
@@ -0,0 +1,9 @@
+.govuk-header {
+  border-bottom: 10px $govuk-brand-colour solid;
+
+  &__container {
+    // Doesn't seem to be any need to do this but leaving it in case there's
+    // any strange bugs in other browsers in the future
+    /* border-bottom: 0; */
+  }
```

Comment:
Uhh, if there is no need for it, I'd much rather not have a comment saying that.

Reply:
There are two borders overlaid over each other right now - that's strange and unexpected behaviour, this serves as documentation for that. A future update of govuk-frontend could break it, or there could be issues in browsers we haven't tested yet.

Equally, removing the other border could break in future updates of govuk-frontend too.

I think they're both pretty unlikely to happen, but think it's important to write down somewhere what's happening.